west.yml

```diff
@@ -0,0 +1,37 @@
+# The west manifest file for the nRF Connect SDK (NCS).
```

Comment:
This shouldn't be here.

Reply:
we were using CI_LIB to get/control the zephyr-rtos/ci-tools repo before.  I think using west.yml is a better.

Reply:
I also think this is very misleading because you are creating a manifest file that only references ci-tools but not zephyr itself, while MCUboot uses Zephyr to build. If all you need is to hardcode a particular version of ci-tools then you can do that elsewhere, but I agree with @mbolivar that this should not be here, because our users will see a `west.yml` in the root of our MCUboot fork and they will assume that this is a standalone package that can be passed to `west init` in order to retrieve MUCboot and its dependencies (including zephyr) and to build them. This won't be the case, so it's abusing west.

Reply:
ok, I will remove and get ci-tools a different way.